.github/workflows/test.yml

```diff
@@ -0,0 +1,44 @@
+name: Test suite
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Commented sections to be uncommented once CI is active
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    # strategy:
+    #   matrix:
+    #     python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+    #     requires: ["", "requirements.txt"]
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      # - name: Set up Python ${{ matrix.python-version }}
+      #   uses: conda-incubator/setup-miniconda@v2
```

Comment:
I'm going to try using miniconda instead of the standard Python setup, as conda still allows Python 2 to be installed. It will also allow us to install the FSL tools needed.

Reply:
See https://github.com/Washington-University/gradunwarp/pull/12 for removing the fsl dependency, it is only used for fslval / fslorient.